pygfx/renderers/wgpu/lineshader.py

```diff
@@ -113,6 +113,59 @@ def __init__(self, wobject):
             pass  # TODO: do benchmarks and enable if it makes things faster
             # self["line_type"] = "quickline"
 
+        # Handle dashing
+        if material.dash_pattern:
+            # Set dash props
+            self["dashing"] = True
+            self["dash_pattern"] = tuple(wobject.material.dash_pattern)
+            self["dash_count"] = len(wobject.material.dash_pattern) // 2
+            # For line segments we can calculate the distance between nodes in the shader.
+            # For normal lines, we need a cumulative distance.
+            if not isinstance(material, LineSegmentMaterial):
```

Comment:
and LineThinSegmentMaterial right?

Reply:
The thin lines don't do dashing at all. But we might indeed need a check for it somewhere around here. Should probably add a validation example for thin lines.

Reply:
That makes sense.